src/cpu/aarch64/acl_thread.cpp

```diff
@@ -65,8 +79,37 @@ void acl_set_threadpool_num_threads() {
         });
     }
 }
+// Swap BenchmarkScheduler for custom scheduler builds (i.e. ThreadPoolScheduler)
+void acl_set_benchmark_scheduler_tp() {
+    static std::once_flag flag_once;
+    // Create threadpool scheduler
+    std::unique_ptr<arm_compute::IScheduler> threadpool_scheduler
+            = std::make_unique<ThreadpoolScheduler>();
+    arm_compute::IScheduler *_real_scheduler = nullptr;
+    _real_scheduler = threadpool_scheduler.release();
+    // Create benchmark scheduler and set TP as real scheduler
+    std::shared_ptr<arm_compute::IScheduler> benchmark_scheduler
+            = std::make_unique<BenchmarkScheduler>(*_real_scheduler);
+    std::call_once(flag_once,
+            [&]() { arm_compute::Scheduler::set(benchmark_scheduler); });
+}
 #endif
 
+void set_acl_threading() {
+#if DNNL_CPU_THREADING_RUNTIME == DNNL_RUNTIME_OMP
+    acl_thread_bind();
+    if (verbose_has_create_profile()) { acl_set_benchmark_scheduler_default(); }
+#endif
+#if DNNL_CPU_THREADING_RUNTIME == DNNL_RUNTIME_THREADPOOL
+    if (verbose_has_create_profile()) {
+        acl_set_benchmark_scheduler_tp();
+    } else {
+        acl_set_threadpool_scheduler();
```

Comment:
(nit) maybe align naming scheme?
For example `acl_set_scheduler_tp/acl_set_scheduler_tp_benchmark`